### DISCUSSION TITLE
Backpfeifengesicht is a German word that describes what?

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [ParanoidUser/yolo#658] [Galaxy Trivia] [answered]
Answer with an image

**Comment by ParanoidUser** [comment, marked as answer]
Great questions, **Jakob**! 👏👏 Thank you for your active participation in the project. Keep them coming!

---
![image](https://user-images.githubusercontent.com/5120290/235360614-ed97e792-a675-40f2-886c-1d8ea358281b.png)

**Comment by peterbaccaro** [comment]
"Backpfeifengesicht" is a German word that roughly translates to "a face in need of a slap." The term is used to describe someone whose face looks like it deserves to be slapped, usually due to their unpleasant or irritating demeanor. The word is a combination of "backpfeife," which means "slap," and "gesicht," which means "face."

![slap](https://user-images.githubusercontent.com/74666079/235365624-1ed10a1b-4043-4ed0-83a5-6d1fd137eb7e.jpg)
